source/query-bar.txt

```diff
@@ -8,36 +8,36 @@ From the various Collections tabs, you can use the query bar to specify
 MongoDB queries.  For details, see :ref:`querying`.
 
 .. figure:: /images/compass/query-bar-documents-view.png
-   :figwidth: 780px
+   :figwidth: 805px
 
-You can specify a query filter in the query bar. If you click the 
+You can specify a query filter in the query bar. If you click 
 :guilabel:`Options` to expand the query bar, you also have access to 
 the ``project``, ``sort``, ``skip``, and ``limit`` options of the query.
 
 .. list-table::
    :widths: 15 85
 
-   * - ``Filter``
+   * - Filter
 
      - Filter for the schema view.
 	 
-   * - ``Project``
+   * - `Project <https://docs.mongodb.com/manual/reference/operator/aggregation/project/>`_
```

Comment:
@kay-kim , links updated at https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-748/query-bar.html. Ready for another look.